filament/src/details/Engine.h

```diff
@@ -126,19 +126,25 @@ class FEngine : public Engine {
     static constexpr size_t CONFIG_FROXEL_SLICE_COUNT      = 16;
     static constexpr bool   CONFIG_IBL_USE_IRRADIANCE_MAP  = false;
 
-    static constexpr size_t CONFIG_PER_RENDER_PASS_ARENA_SIZE   = filament::CONFIG_PER_RENDER_PASS_ARENA_SIZE;
-    static constexpr size_t CONFIG_PER_FRAME_COMMANDS_SIZE      = filament::CONFIG_PER_FRAME_COMMANDS_SIZE;
-    static constexpr size_t CONFIG_MIN_COMMAND_BUFFERS_SIZE     = filament::CONFIG_MIN_COMMAND_BUFFERS_SIZE;
-    static constexpr size_t CONFIG_COMMAND_BUFFERS_SIZE         = filament::CONFIG_COMMAND_BUFFERS_SIZE;
+    // Creation parameters
+    size_t mMinCommandBufferSize;           // minimum size of command buffer (in bytes)
+    size_t mCommandBufferSize;              // size of command buffer (in bytes)
+    size_t mPerFrameCommandsSize;           // size of the high-level draw commands buffer (in bytes)
+    size_t mPerRenderPassArenaSize;         // size of the per-pass arena buffer (in bytes)
+    size_t mDriverHandleArenaSize;          // size of driver handle arena (in bytes)
```

Comment:
move these in the `private` section towards the bottom (with all the other private fields)

Reply:
Fixed in my next update.